### DISCUSSION TITLE
How can I download WW3 Archived Data in MMBA?

### DISCUSSION THREAD

**Discussion by ZHONG-YICHEN** [NOAA-EMC/WW3#1135] [General]
I want download recent years WW3 simulation data, and I find Data Access in this Web: https://polar.ncep.noaa.gov/waves/ensemble/download.shtml
While, when I tried to follow the prompts and access the site ftp://polar.ncep.noaa.gov/pub/history/waves, I got nothing.
I really want to get the WW3 archived data from 2018-2023 as high precision as possible... Because I am struggling a ddl about a climate project. 
Is there any other way I can get WW3 simulation data?

**Comment by JessicaMeixner-NOAA** [comment]
@ricampos can point you to some work he's done with an ensemble reforecast.  Otherwise from NCEP, we only have the last 5 days of operational data available publicly.

**Comment by ricampos** [comment]
Using AWS is an easy way to have access. For WW3 GFS archive:
https://registry.opendata.aws/noaa-gfs-bdp-pds/
https://noaa-gfs-bdp-pds.s3.amazonaws.com/index.html
For WW3 GEFS archive:
https://registry.opendata.aws/noaa-gefs/
https://noaa-gefs-pds.s3.amazonaws.com/index.html
